README.rst

```diff
@@ -2,20 +2,136 @@
+
+Installation
+============
+
+
+Downloading the binary distribution
+-----------------------------------
+
+Binary (threaded) distribution of the latest stable release can be found on the
+`stable release page <http://www.dftbplus.org/download/dftb-stable/>`_.
+
+
+Building from source
+--------------------
+
+**Note:** This section describes the building with default settings (offering
+only a subset of all possible features in DFTB+). For more detailed information
+on the build customization and the build process, consult the `detailed building
+instructions <INSTALL.rst>`_.
+
+Download the source code from the `stable release page
+<http://www.dftbplus.org/download/dftb-stable/>`_.
+
+You need CMake (>= 3.5.0) to build DFTB+. Create a build folder
+(e.g. ``build``) in the source folder and start the configuration from
+there. Pass your compilers as environment variables (``FC`` and ``CC``) and
+the location where the code should be installed (``-DCMAKE_INSTALL_PREFIX``)::
+
+  mkdir build
+  cd build
+  env FC=gfortran CC=gcc cmake -DCMAKE_INSTALL_PREFIX=$HOME/opt/dftb+ ..
+
+If the configuration was successful, start the build with ::
+
+  make -j
+
+After successful build, you should test the code. First download the SK-files
+needed for the test ::
+
+  cd ..
+  ./utils/get_opt_externals slakos
+  cd build
+
+and then run the tests with ::
+
+  ctest -j
+
+If the tests were successful, install the package with ::
+
+  make install
+
+For further details see the `detailed building instructions <INSTALL.rst>`_.
+
+
+Parameterisations
+=================
+
+In order to carry out calculations with DFTB+, you need according
+parameterisations (a.k.a. Slater-Koster files). You can download them from
+`dftb.org <https://dftb.org>`_.
+
+
+Documentation
+=============
+
+Consult following resources for documentation:
+
+* `Step-by-step instructions with selected examples (DFTB+ Recipes)
+  <http://dftbplus-recipes.readthedocs.io/>`_
+
+* `Reference manual describing all features (DFTB+ Manual)
+  <http://www.dftbplus.org/fileadmin/DFTBPLUS/public/dftbplus/latest/manual.pdf>`_
+
+
+Citing
+======
+
+When publishing results obtained with DFTB+, please cite following works:
+
+* `DFTB+, a software package for efficient approximate density functional theory
+  based atomistic simulations; J. Chem. Phys. 152, 124101 (2020)
+  <https://doi.org/10.1063/1.5143190>`_
+
+* Reference publications of the Slater-Koster parameterization sets you
+  used. (See `dftb.org <https://dftb.org>`_ for the references.)
+
+* Methodological papers relevant to your calculations (e.g. excited states,
+  electron-transport, third order DFTB etc.). Those references can be found in
+  the `DFTB+ manual
+  <http://www.dftbplus.org/fileadmin/DFTBPLUS/public/dftbplus/latest/manual.pdf>`_.
+
+
+Contributing
+============
+
+Welcome in the DFTB+ developer community!
+
+The project is `hosted on github <http://github.com/dftbplus/dftbplus>`_.
```

Comment:
Might be a good idea to give the github.com/dftbplus address as the project, as this then includes the other components of the project, not just DFTB+.